DonWorry/DonWorry/Views/CheckAttendanceViews/CheckAttendanceView.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  SpaceMainView.swift
+//  DonWorry
+//
+//  Created by Hankyu Lee on 2022/06/02.
+//
+//
+import SwiftUI
+
+struct CheckAttendanceView: View {
+    
+    @Environment(\.presentationMode) var mode: Binding<PresentationMode>
+    let leftPaddingSize: CGFloat = 25
+    let cards = [0, 1, 2, 3, 4]
+    private var numberOfCards: Int {
+        cards.count
+    }
+    private var numberOfcheck: Int {
+        checkedArray.count
+    }
+    private func filterArray(_ checkedArray: [Int], index: Int) -> [Int] {
+        if checkedArray.contains(index) {
+            return checkedArray.filter { $0 != index }
+        } else {
+            return checkedArray + [index]
+        }
+    }
```

Comment:
struct 밖에 따로 extension 으로 함수를 빼서 만들어주세요! 
preview아래에!


```suggestion
extension CheckAttendanceView {
    private func filterArray(_ checkedArray: [Int], index: Int) -> [Int] {
        if checkedArray.contains(index) {
            return checkedArray.filter { $0 != index }
        } else {
            return checkedArray + [index]
        }
    }
}


```